.github/CONTRIBUTING.md

```diff
@@ -56,6 +56,24 @@ approved by the reviewer.
 
 Once the pull request is approved, a team member will take care of merging.
 
+## Contributing training scripts
+
+The KerasCV team is working to include a catalog of high-performing model training scripts for the models included in KerasCV.models and KerasCV.applications, and is welcoming contributions for these scripts. These training scripts serve as documentation of good training techniques and will be used to train weights that will be offered in KerasCV models through the package.
+
+The KerasCV team will run submitted training scripts to produce weights for KerasCV, and will attribute strong weights to contributors via a training script ranking system. Stay tuned for the detail of that.
+
+See [this example PR](https://github.com/keras-team/keras-cv/pull/618) which added an initial training script for DenseNets on ImageNet. Please follow the structure of this training script in contributing your own script. New scripts should either:
+- Train a model or application for which we don't have a training script already
+- Include a meaningfully different training approach for a given model (e.g. a specific type of pretraining)
+
+Incremental improvements to existing training scripts are welcome, provided that they come with evidence of improved validation performance.
+
+When contributing training scripts, please include documentation to support decisions about training including hyperparameter choices. Examples of good documentation would be recent literature or a reference to a hyperparameter search.
```

Comment:
Before we merge:

lets have you go through the process (once) of exporting weights, then improving them once, generating a JSON file with the history, etc

Reply:
I'm working on this now -- I'm not sure about the history bit quite yet but at least the weight exporting bit needs to be ready.

Reply:
I think tracking history in a JSON file is about P1 - true it is less important than offering weights but it would be a pain to retroactively create it.